cpu/native/include/trace.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    trace       Stack traceback (only under native)
+ * @ingroup     core_util
+ * @brief       Address-trace back.
+ *
+ * If you add @ref trace_print() function to your implementation a stack
```

Comment:
The documentation is a bit confusing. It sounded to me like one would have to implement the function. I guess the following would have been less confusing for me:
`When you call the @ref trace_print() function a stack`

Reply:
Addressed.